src/contents/en-gb/doc_xamarin.md

```diff
@@ -0,0 +1,210 @@
+# SAFE Mobile App Tutorial
+
+In this tutorial, we will create a [Xamarin.Forms](https://github.com/xamarin/Xamarin.Forms/) application. Xamarin.Forms is an open source and cross-platform tool which provides a way to quickly build native apps for iOS, Android, Windows and macOS using C# and F#.
+
+To create this app, we will use the [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) NuGet package which exposes the `safe_app` API to connect and interact with the SAFE Network.
+
+Download demo code for this tutorial from [here](https://github.com/maidsafe/safe-getting-started-dotnet/SafeTodoExample).
+
+## Pre-requisites
+
+Before we start working on our first SAFE app, make sure you have the following tools installed to work with this tutorial.
+
+- **Visual Studio with Xamarin**: You need Visual Studio (any edition) to build our first SAFE Network mobile application. Follow the instructions [here](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/get-started/installation) to download and install visual studio with Xamarin.Forms for your Operating System.
+- **SafeAuthenticator**: Safe Authenticator mobile application will be required to initiate the authentication process with SAFE Network. Please follow installation instruction [here](https://github.com/maidsafe/safe_mobile#Installation).

```

Comment:
The word *instruction* should be *instructions*
The word *here* is black - however other links on the page are blue to indicate they have links. 
Following the link, I navigate to the safe.authenticator.ios repo - there is no *read me* file with download instructions